items/resources.py

```diff
@@ -79,6 +79,10 @@ def delete_detail(self, object_list, bundle):
     return self.check_access(bundle) 
 
 
+class TrashAuthorization(Authorization):
+  pass
```

Comment:
if it's empty, maybe you can omit `authorization = TrashAuthorization()` and remove the dummy?